expression/builtin_encryption.go

```diff
@@ -228,7 +228,7 @@ func (b *builtinPasswordSig) evalString(row []types.Datum) (d string, isNull boo
 	sc := b.ctx.GetSessionVars().StmtCtx
 	pass, isNull, err := b.args[0].EvalString(row, sc)
 	if isNull || err != nil {
-		return "", false, errors.Trace(err)
+		return "", err != nil, errors.Trace(err)
```

Comment:
Should be `return "", true, errors.Trace(err)`?

Reply:
no, `password(NULL)` should return an empty string, not `NULL`